README.md

```diff
@@ -1,23 +1,176 @@
 # ModelDB: A system to manage ML models
 **Website**: [http://modeldb.csail.mit.edu](http://modeldb.csail.mit.edu)
 
+**See the ModelDB frontend in action**: [Watch the video](https://youtu.be/m2PhgEkx8-E)
+
+## Contents
+
+- [Overview](#overview)
+- [News](#news)
+- [Setup and Installation](#setup-and-installation)
+- [Usage and Samples](#usage-and-samples)
+    - [Incorporate ModelDB into your ML workflow](#incorporate-modeldb-into-your-ml-workflow)
+    - [View your models in ModelDB](#view-your-models-in-modeldb)
+- [Documention](#documentation)
+- [Contact Us](#contact-us)
+
+## Overview
 ModelDB is an end-to-end system to manage machine learning models. It can ingest models and metadata as they are built, store this data in a structured format, and surface it through a web-frontend for rich querying.
 
-<img src="docs/getting_started/images/frontend-1.png" width="70%"><br>
-ModelDB frontend
+ModelDB's frontend shows summaries and graphs of metrics where users can filter on configurations and metrics, configure the x- and y- axes, group by certain configurations, among others. The visualizations and filtering allow for a better experience when comparing models.
 
+<img src="docs/getting_started/images/frontend-1.png" width="90%"><br>
+<img src="docs/getting_started/images/frontend-2.png" width="90%"><br>
+<img src="docs/getting_started/images/frontend-4.png" width="90%"><br>
 
+ModelDB **works with any machine learning workflow** with ModelDB's Light API. Model metrics and metadata can be synced by calling a few functions just like in [this sample basic workflow](client/python/samples/basic/BasicWorkflow.py). The information can also be synced all at once from a JSON or a YAML file as shown in [this sample](client/python/samples/basic/BasicSyncAll.py).
```

Comment:
Pull out supported environments more explicitly and at the top level. E.g. "All environments via Light API, advanced support for sklearn and spark.ml"